basic_gdeploy_doc.md

```diff
@@ -0,0 +1,298 @@
+#### Introduction of gdeploy and how to use gdeploy to configure glusterfs
+-----
+
+This document is intended to give a basic overview of the gdeploy tool like,
+what is gdeploy, what does it do, and how you can use it.
+
+##### What is glusterfs?
+
+GlusterFS is a scale-out network-attached storage file system. It has found applications 
+including cloud computing, streaming media services, and content delivery networks. GlusterFS 
+was developed originally by Gluster, Inc., then by Red Hat, Inc., after their purchase of 
+Gluster in 2011.
+
+##### What is gdeploy?
+
+gdeploy is an automation tool that uses Ansible to simplify setting up `glusterfs` 
+file system for known configurations. You can easily deploy and configure glusterfs 
+software across multiple machines using gdeploy tool. You must install `ansible version 1.9.2` 
+or higher to use gdeploy.
+
+Or
+
+
+It's a python based automation tool that can be used to deploy and configure
+`glusterfs` file system across multiples machines. Gdeploy must be run from a
```

Comment:
Just mention it is an automation tool that uses ansible to automate the deployments. Leave out python based automation tool.